File: python/Day_2/Question_06.py
str1 = input('string 1: ')
str2 = input('string 2: ')

def checkAmagam(str1, str2):
    str1 = str1.replace(" ","").lower()
    str2 = str2.replace(" ","").lower()

    if(sorted(str1) == sorted(str2)):
        print('Your string is anagram')
    else:
        print('Your string is not anagram')

result = checkAmagam(str1, str2)

File: javascript/Day_2/Question_06.js
let str1 = "listen";
let str2 = "silent";

const checkAmagam = (st1, str2) => {
  str1 = str1.toLowerCase().split("").sort().join("");
  str2 = str2.toLowerCase().split("").sort().join("");

  if (str1 === str2) {
    console.log("The strings are anagrams.");
  } else {
    console.log("The strings are not anagrams.");
  }
};

checkAmagam(str1, str2);
